crates/marine-test-macro-impl/src/marine_test/token_stream_generator/service_generator.rs

```diff
@@ -23,10 +23,12 @@ use crate::marine_test::modules_linker::LinkedModules;
 use super::service_generation_utils::generate_service_definition;
 
 use marine_it_parser::it_interface::IModuleInterface;
+
```

Comment:
Shouldn't we group marine_it_parser with rest of non-local crates?

Reply:
looks like yes